### DISCUSSION TITLE
[Feature Request] Unable to use a custom tailwind prefix with NextUI

### DISCUSSION THREAD

**Discussion by Moe03** [heroui-inc/heroui#4693] [Feature requests]
### Is your feature request related to a problem? Please describe.

Unable to use a custom prefix with NextUI, not a major bug but necessary if NextUI is used with web builders like wordpress or webflow where you must address the css issue, also an option to just hash all the classnames on build would be absolutely awesome.

### Describe the solution you'd like

Just adding support for the prefix attribute by default in tailwind, or in NextUI adding a way to hash all classnames on build would also be great.

### Describe alternatives you've considered

Tried to do this myself but it was pain, thought it would just be there by default

**Comment by Canglangzhuoxi** [comment]
Great library! Thanks for your work! I like this library, but I can't use it in production for this reason

**Comment by wingkwong** [comment]
@Canglangzhuoxi have you tried just passing the prefix to nextui? ref: https://nextui.org/docs/customization/colors#css-variables

```tsx
 plugins: [
  nextui({
    prefix: "myapp"
  }),
],
```

**Comment by C4T4** [comment]
same problem 

`  nextui({
    prefix: "myapp"
  }),`

is not working
